src/helpers/tests/dimensions.spec.ts

```diff
@@ -981,3 +985,56 @@ describe("getGeneralDimensionsFromAFM", () => {
         expect(getGeneralDimensionsFromAFM(afm)).toEqual(expectedDimensions);
     });
 });
+
+describe("getGeoChartDimensions", () => {
+    function createBuckets(pushpinProps: IGeoPushpinChartBucketProps): VisualizationObject.IBucket[] {
```

Comment:
should we re-use the `getBuckets` function from `src/components/GeoPushpinChart.tsx`?

Reply:
i just needs mock-data, so don't need to import getBuckets from geoPushpinchart component.